examples/md/slides.md

```diff
@@ -44,7 +44,7 @@ Typography
 ```js
 class SuperCoolComponent extends React.Component {
   render() {
-    return <p>code slide works in markdown too whaaaaat</p>;
+    return <p>code slide works in markdown too whaaaaat</p><p>code slide works in markdown too whaaaaat</p><p>code slide works in markdown too whaaaaat</p>;
```

Comment:
NICE -- this looks great for the open issues we have for:

* MD code indentation bug: #898 
* Copy and pasteable code blocks: #928

Reply:
say _whaaaaat_? 😛 Let me know if you want me to do any code reviews or toss it through my use case when it's ready to go.